src/sysfs_fuse.c

```diff
@@ -57,7 +57,7 @@ static int do_cpuset_read(char *cg, char *buf, size_t buflen)
                 use_view = false;
 
         if (use_view)
-                max_cpus = max_cpu_count(cg);
+                max_cpus = max_cpu_count(cg, cg);
```

Comment:
Do you have any specific reason not to pass `cpuset` and `cpu` cgroup separately here too?

Reply:
thanks for your review and advices.
At first I thought the func named `do_cpuset_read`, maybe it's not good to add `cpu_cg` argument,  it may kind of  break abstract layer. 

I've make some changes , please take a review again.